.stylelintrc

```diff
@@ -6,4 +6,12 @@
     "indentation": 2,
     "unit-whitelist": ["px", "em", "%", "rem", "ms", "s", "vh"],
   }
+<<<<<<< HEAD
+<<<<<<< HEAD
 }
+=======
+}
+>>>>>>> ch(styling): Create scss file for colours
+=======
+}
+>>>>>>> ft(HeaderCard): Create the header card components
```

Comment:
Please remove the conflict markers from the above

Reply:
Thank you for your observation, i will make the changes immediately.